src/item.cpp

```diff
@@ -4569,11 +4569,11 @@ int item::processing_speed() const
     return 1;
 }
 
-bool item::process_food( player * /*carrier*/, point pos )
+bool item::process_food( player * /*carrier*/, const tripoint &pos )
 {
     // TODO: this functions (and all the other process functions) should be called with a tripoint
     // If this gets implemented, don't forget that calc_rot expects an *absolute* position.
```

Comment:
This comment needs to be removed or at least be changed, the first part of it is now completely pointless.